kuka_external_control_sdk_examples/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>kuka_external_control_sdk_examples</name>
```

Comment:
Are we also including the examples?

Reply:
this was needed for the industrial ci, as it tried to build all packages, but this failed unless I specified in the package.xml, that it depends on the sdk package